### PR TITLE
Wrong image URL(Image not loaded ) --typo

### PR DESCRIPTION
**Pull request by partheee**
Wrong image URL (Image not loaded)--typo

### CODE REVIEW

**Review by bellabf — APPROVED** [review]
Thank you for taking care of it

**Review by mmhamdy — APPROVED** [review]
Cool! Thanks

**Review by merveenoyan — APPROVED** [review]
Thanks!

**Comment by merveenoyan** [comment]
do not merge -- let's see the deployed version

File: chapters/en/unit4/multimodal-models/a_multimodal_world.mdx [hunk 1]
```diff
@@ -26,7 +26,7 @@ _An infographic on multimodality and why it is important to capture the overall
 Many times communication between 2 people gets really awkward in textual mode, slightly improves when voices are involved but greatly improves when you are able to visualize body language and facial expressions as well. This has been studied in detail by the American Psychologist, Albert Mehrabian who stated this as the 7-38-55 rule of communication, the rule states:
 "In communication, 7% of the overall meaning is conveyed through verbal mode (spoken words), 38% through voice and tone and 55% through body language and facial expressions."
 
-![Funny Image + Text Meme example](https://huggingface.co/datasets/hf-vision/course-assets/main/resolve/multimodal_fusion_text_vision/bigbang.jpg)
+![Funny Image + Text Meme example](https://huggingface.co/datasets/hf-vision/course-assets/resolve/main/multimodal_fusion_text_vision/bigbang.jpg)
```
Comment (merveenoyan) [inline chapters/en/unit4/multimodal-models/a_multimodal_world.mdx:29]: TBH maybe let's remove this meme. might be subjected to copyright
```suggestion
```

**Review by merveenoyan — APPROVED** [review]
thanks

**Pull request merged**